awskinesis/main.go

```diff
@@ -207,7 +207,7 @@ func (clients Clients) sendCognitoEvent(record *kinesis.Record, shardID *string)
 
 	event := cloudevents.Event{
 		Context: cloudevents.EventContextV03{
-			Type:            "AWS Kinesis Record",
+			Type:            "com.amazon.kinesis",
```

Comment:
Do you view Kinesis as the event producer or more like an event transport? If it's more like a transport, then I'm not sure this is the right value since it doesn't tell you anything about the event itself.

Reply:
I guess it is more like event producer here.